optimization/optimizer.py

```diff
@@ -3,14 +3,15 @@
 from scipy.optimize import minimize
 import argparse
 
+
 global car
 car = Car()
 
 parser = argparse.ArgumentParser(description="Add parameters to model")
-parser.add_argument("-d", "--distance", type=int, required=True, help="Distance to travel (m)")
-parser.add_argument("-t", "--time", type=int, required=True, help="Maximum allowable time (s)")
-parser.add_argument("-v", "--min_velocity", type=int, required=True, help="Minimum allowable velocity (m/s)")
-parser.add_argument("-s", "--step", type=int, required=True,help="Distance between elevation profile measurements")
+parser.add_argument("-d", "--distance", type=int, default=5490, help="Distance to travel (m)")
```

Comment:
Why did you remove the required?

Reply:
Required wouldn't allow the use of default values. Now when no parameters are entered it just uses the default instead.